<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroItens</class>
 <widget class="QDialog" name="cadastroItens">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela de Cadastro de Itens</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>10</y>
     <width>861</width>
     <height>551</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_precoItem">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_criarItem">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomeItem">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_quantidadeItem">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_descricaoItem">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_nome">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome do Item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_preco">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Preço por Unidade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_quantidade">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantidade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_descricao">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição do Item</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Sair</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
